docs/docs/cmd/planner/roster/roster-member-add.md

```diff
@@ -0,0 +1,85 @@
+# planner roster member add
+
+Adds a user to a Microsoft Planner Roster
+
+## Usage
+
+```sh
+m365 planner roster member add [options]
+```
+
+## Options
+
+`--rosterId <rosterId>`
+: ID of the Planner Roster.
+
+`--userId [userId]`
+: User's Azure AD ID. Specify either `userId`, `userName` but not both.
+
+`--userName [userName]`
+: User's UPN (user principal name, e.g. johndoe@example.com). Specify either `userId`, `userName` but not both.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+!!! attention
+    This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+## Examples
+
+Adds a specific user by user name as member of the Planner Roster
+
+```sh
+m365 planner roster member add --rosterId tYqYlNd6eECmsNhN_fcq85cAGAnd --userName john.doe@contoso.com
+```
+
+Adds a specific user by ID as member of the Planner Roster
+
+```sh
+m365 planner roster member add --rosterId tYqYlNd6eECmsNhN_fcq85cAGAnd --userId d049a857-f1c3-4fb3-a629-d8cfb3bd7275
+```
+
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+      "id": "b3a1be03-54a5-43d2-b4fb-6562fe9bec0b",
+      "userId": "2056d2f6-3257-4253-8cfc-b73393e414e5",
+      "tenantId": "5b7b813c-2339-48cd-8c51-bd4fcb269420",
+      "roles": []
+    }
+    ```
+
+=== "Text"
+
+    ```text
+    id      : b3a1be03-54a5-43d2-b4fb-6562fe9bec0b
+    roles   : []
+    tenantId: 5b7b813c-2339-48cd-8c51-bd4fcb269420
+    userId  : 2056d2f6-3257-4253-8cfc-b73393e414e5
+    ```
+
+=== "CSV"
+
+    ```csv
+    id,userId,tenantId,roles
+    b3a1be03-54a5-43d2-b4fb-6562fe9bec0b,2056d2f6-3257-4253-8cfc-b73393e414e5,5b7b813c-2339-48cd-8c51-bd4fcb269420,[]
+    ```
+    
+=== "Markdown"
+
+    ```md
+    # planner roster add
```

Comment:
Seems like this is a response from another command

Reply:
indeed, should be:
`# planner roster member add --rosterId "e6fmvM_yi0OJgvmepE5uj5cAE6qX" --userId "2056d2f6-3257-4253-8cfc-b73393e414e5"`